### DISCUSSION TITLE
Impossible to install Ptero exporter

### DISCUSSION THREAD

**Discussion by Tr3yWay996** [LOENS2/pterodactyl_exporter#40] [Q&A] [answered]
The python 3.11 image It's supposed to be pulling isn't accessible, so uhh, what am I supposed to do ??
Here is the supposed image it's gonna pull, not accessible as you can see: https://ghcr.io/v2/software-noob/pterodactyl-images/manifests/python_3.11

And this is the log, stating it cant access it obviously
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: Checking server disk space usage, this could take a few seconds...
[Pterodactyl Daemon]: Updating process configuration files...
[Pterodactyl Daemon]: Ensuring file permissions are set correctly, this could take a few seconds...
container@pterodactyl~ Server marked as starting...
[Pterodactyl Daemon]: Pulling Docker container image, this could take a few minutes to complete...
container@pterodactyl~ Server marked as offline...
container@pterodactyl~ Error Event [1b7b764a-132d-4b32-8532-0082233d064b]: environment/docker: failed to run pre-boot process: environment/docker: failed to pull "ghcr.io/software-noob/pterodactyl-images:python_3.11" image for server: Error response from daemon: Head "https://ghcr.io/v2/software-noob/pterodactyl-images/manifests/python_3.11": denied
[Pterodactyl Daemon]: Finished pulling Docker container image
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Checking server disk space usage, this could take a few seconds...
[Pterodactyl Daemon]: Updating process configuration files...
[Pterodactyl Daemon]: Ensuring file permissions are set correctly, this could take a few seconds...
container@pterodactyl~ Server marked as starting...
[Pterodactyl Daemon]: Pulling Docker container image, this could take a few minutes to complete...
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: Server crash was detected but an error occurred while handling it.
[Pterodactyl Daemon]: Finished pulling Docker container image
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Aborting automatic restart, last crash occurred less than 60 seconds ago.

**Comment by LOENS2** [comment]
That is not supposed to be that way. This seems to be a problem with the base image I use for the egg installation of the pterodactyl exporter. The repository for that was archived on Nov. 8: https://github.com/Software-Noob . I have to find a new base image, that may take a while. I'll respond as soon as I have found a solution.

**Comment by LOENS2** [comment, marked as answer]
Should be fixed. I replaced the base image with ghcr.io/parkervcp/yolks:python_3.11. Please update the egg and reinstall the server afterwards.

**Reply by Tr3yWay996** [reply]
> Should be fixed. I replaced the base image with ghcr.io/parkervcp/yolks:python_3.11. Please update the egg and reinstall the server afterwards.

Hey, thanks for the very quick reply, it works now thanks
Do you know how I could get a client API key so it can monitor my servers now, tho ? I made an app API key, and it asks a client one, but there is no setting to actually make one, I asked in the discord server of pterodactyl, but I still don't have any answers

**Reply by Tr3yWay996** [reply]
Very good support, even tho I should have looked before asking instantly for the API key part since i found my answer
